syne_tune/optimizer/schedulers/searchers/gp_multifidelity_searcher.py

```diff
@@ -304,15 +246,6 @@ def evaluation_failed(self, trial_id: str):
         self.state_transformer.mark_trial_failed(trial_id)
 
     def cleanup_pending(self, trial_id: str):
```

Comment:
Why are we removing this docstring? Is it because its not user-facing and we dont want it to appear in documentation?